training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -276,21 +277,21 @@ object AdditiveModelTrainer {
                       label: Double,
                       params : AdditiveTrainerParams) : Double = {
     val flatFeatures = Util.flattenFeatureWithDropout(fv, params.dropout)
-    val denseFeatures = Util.denseFeatureDropout(fv, params.dropout)
+    val denseFeatures = MultiDimensionSpline.featureDropout(fv, params.dropout)
```

Comment:
perhaps add a comment above saying this denseFeatures is for MultiDimensionSpline for generic denseFeatures